File: src/main/java/org/lineageos/sleepaudio/ui/SleepAudioFragment.java
package org.lineageos.sleepaudio.ui;

import android.os.Bundle;
import androidx.preference.PreferenceFragment;
import org.lineageos.sleepaudio.R;

public class SleepAudioFragment extends PreferenceFragment {
    @Override
    public void onCreatePreferences(Bundle savedInstanceState, String rootKey) {
        addPreferencesFromResource(R.xml.sleep_audio_settings);
    }
}

File: app/src/main/kotlin/org/lineageos/sleepaudio/ui/SleepAudioFragment.kt
package org.lineageos.sleepaudio.ui

import android.os.Bundle
import androidx.preference.PreferenceFragmentCompat
import org.lineageos.sleepaudio.R

class SleepAudioFragment : PreferenceFragmentCompat() {
    override fun onCreatePreferences(savedInstanceState: Bundle?, rootKey: String?) {
        setPreferencesFromResource(R.xml.sleep_audio_settings, rootKey)
    }
}
